### PR TITLE
Issue635 dx water coooled

### PR DESCRIPTION
**Pull request by mwetter**
Merge the water-cooled dx coil

### CODE REVIEW

File: Buildings/Fluid/HeatExchangers/DXCoils/WaterCooled/Examples/VariableSpeed.mo [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          mCon_flow_nominal=1,
+          TEvaIn_nominal=273.15+26.67,
+          TConIn_nominal=273.15+29.4),
+        perCur=Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.Examples.PerformanceCurves.Curve_I())}) "Coil data"
+    annotation (Placement(transformation(extent={{60,60},{80,80}})));
+  Modelica.Blocks.Sources.Ramp     mCon_flow(
+    duration=600,
+    startTime=6000,
+    height=0,
+    offset=1)                                     "Condensor inlet mass flow"
```
Comment (mwetter) [inline Buildings/Fluid/HeatExchangers/DXCoils/WaterCooled/Examples/VariableSpeed.mo]: Correct formatting in this file (and maybe in other files as well that have superfluous empty spaces or line breaks.

File: Buildings/Fluid/HeatExchangers/DXCoils/WaterCooled/package.mo [hunk 2]
```diff
@@ -0,0 +1,4 @@
+within Buildings.Fluid.HeatExchangers.DXCoils;
+package WaterCooled "Waterr-cooled DX package unit"
+
```
Comment (mwetter) [inline Buildings/Fluid/HeatExchangers/DXCoils/WaterCooled/package.mo]: The `package.mo` files need an info section that describes the package.

File: Buildings/Fluid/Interfaces/TwoPortHeatMassExchanger.mo [hunk 3]
```diff
@@ -96,7 +96,7 @@ equation
       points={{1,0},{100,0}},
       color={0,127,255}));
   connect(port_a, preDro.port_a) annotation (Line(
-      points={{-100,0},{-90,0},{-90,0},{-80,0},{-80,0},{-60,0}},
+      points={{-100,0},{-90,0},{-80,0},{-60,0}},
```
Comment (mwetter) [inline Buildings/Fluid/Interfaces/TwoPortHeatMassExchanger.mo]: Please revert this change as it may otherwise give conflict with other branches.

File: Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/DoubleSpeed/package.mo [hunk 4]
```diff
@@ -0,0 +1,23 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Data;
+package DoubleSpeed "Performance data for DoubleSpeed DXCoils"
+import Buildings;
```
Comment (mwetter) [inline Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/DoubleSpeed/package.mo]: Please remove the `import Buildings;` statement

File: Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/Generic/BaseClasses/PerformanceCurve.mo [hunk 5]
```diff
@@ -36,15 +36,8 @@ record PerformanceCurve "Data record for a performance curve"
...
-This record declares the data used to specify performance curves for DX coils.
-</p>
-<p>
-See the information section of
-<a href=\"modelica://Buildings.Fluid.HeatExchangers.DXCoils.Data.Generic.DXCoil\">
-Buildings.Fluid.HeatExchangers.DXCoils.Data.Generic.DXCoil</a>
-for a description of the data.
-</p>
+<p>This record declares the data used to specify performance curves for air-cooled DX coils. </p>
+<p>See the information section of <a href=\"modelica://Buildings.Fluid.HeatExchangers.DXCoils.Data.Generic.DXCoil\">Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Data.Generic.DXCoil</a> for a description of the data. </p>
```
Comment (mwetter) [inline Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/Generic/BaseClasses/PerformanceCurve.mo]: Please add a line break as otherwise version control becomes hard if lines are too long.

File: Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/Generic/BaseClasses/package.mo [hunk 6]
```diff
@@ -0,0 +1,37 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Data.Generic;
+package BaseClasses "Package with base classes for performance data"
+  extends Modelica.Icons.BasesPackage;
+
```
Comment (mwetter) [inline Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Data/Generic/BaseClasses/package.mo]: Please remove empty lines. Sometimes Dymola seems to add them.

**Comment by YangyangFu** [comment]
@mwetter I've updated the code based on your initial review.

**Comment by mwetter** [comment]
@YangyangFu Thanks for the revisions. I will merge it and further review it.
